package.json

```diff
@@ -55,7 +55,7 @@
     "refresh-db": "./refresh-db.sh",
     "seed-usa": "yarn build && node build/db/import/usa/USADay0Seed.js",
     "init-db": "yarn build && node build/db/utils/jobs/UpdateAllRunner.js",
-    "update-search": "node build/db/export/Typesense/Typesense.js"
+    "update-search": "tsc ; node build/db/export/Typesense/Typesense.js"
```

Comment:
can we have a separate job?  Ex:  `yarn update-search-climbs` and `yarn update-search-areas`?  This way we can test and run each job independently without impacting one another.

Reply:
Sure, I'll add that in